src/components/LocationDropdown.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Fragment, useState } from 'react'
+import { Listbox, Transition } from '@headlessui/react'
+import { LOCATIONS } from '../constants/locations';
+
+
+export type LocationType = {
+  country: string;
+  value: number;
+}
+
+type LocationDropdownType = {
+  selectedLocation?: LocationType;
+  setSelectedLocation: (location: LocationType) => void;
+}
+
+export const LocationDropdown = ({ selectedLocation, setSelectedLocation }: LocationDropdownType) => {
+
+  return (
+    <div className="fixed top-16 w-72">
+      <Listbox value={selectedLocation} onChange={setSelectedLocation}>
+        <div className="relative mt-1">
+          <Listbox.Button className="relative w-full cursor-default rounded-lg bg-white py-2 pl-3 pr-10 text-left shadow-md focus:outline-none focus-visible:border-indigo-500 focus-visible:ring-2 focus-visible:ring-white focus-visible:ring-opacity-75 focus-visible:ring-offset-2 focus-visible:ring-offset-orange-300 sm:text-sm">
```

Comment:
I am not sure we need the styling since none of its used and we might want to go down a different design route? What are your thoughts?